lib/Dancer/Plugin/SimpleCRUD.pm

```diff
@@ -710,12 +721,23 @@ register_hook(qw(
 ));
 register_plugin;
 
+
+sub _database {
+    my %args = @_;
+    if ($args{db_connection_provider} && $args{db_connection_provider} eq "DBIC") {
+        require Dancer::Plugin::DBIC;
+        return Dancer::Plugin::DBIC::schema($args{db_connection_name})->storage->dbh;
+    } else {
```

Comment:
I think I'd prefer it to squawk loudly if `db_connection_provider` is set, but to a value we don't recognise - e.g. someone passed `DBIX` or something, and wonders why it was silently ignored and D::P::D used anyway.

Reply:
agreed, fixed